lib/client/db/dbcmd/dbcmd.go

```diff
@@ -143,20 +151,32 @@ func (c *cliCommandBuilder) getConnectCommand() (*exec.Cmd, error) {
 	return nil, trace.BadParameter("unsupported database protocol: %v", c.db)
 }
 
+func (c *cliCommandBuilder) GetRelativeConnectCommand() (*exec.Cmd, error) {
```

Comment:
Nit: Add godoc.

Also, I'm not sure that "relative" is the correct word here but I'm not sure what a better one would be. Maybe just `GetConnectCommandNoAbsPath` or something? Or just push this logic to the caller instead of a method here.